.github/workflows/continuous-integration.yml

```diff
@@ -51,3 +51,9 @@ jobs:
     - name: Run all checks
       run: |
         nix flake check --print-build-logs
+
+    - name: Compile and typecheck benchmarks
+      run: |
+        nix develop --accept-flake-config
+        cargo bench --no-run
+        find benches -type f -name "*.ncl" -print0 | xargs -0 -I file nix run --accept-flake-config . -- -f file typecheck
```

Comment:
A neater way to do this would be adding a `devShells.ci` to the flake with `nickel` in its `buildInputs`, so that we could then just pass `nickel` to `xargs`.